analytics.go

```diff
@@ -94,3 +151,45 @@ func (c *Client) getAggregatedData(ctx context.Context, analytics AnalyticsReque
 
 	return analyticsResponse, nil
 }
+
+// GetRawDataSingleIncident gets the raw analytics for the requested incident.
+func (c *Client) GetRawDataSingleIncident(ctx context.Context, id string) (RawData, error) {
```

Comment:
Could we update the signatures to return a pointer to the result? e.g. `func (c *Client) GetRawDataSingleIncident(ctx context.Context, id string) (*RawData, error)`